script/release.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Does this script require any bash specific functionality? 
Maybe you can change this to `#!/bin/sh`